### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by trancer1994**

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -3564,3 +3564,21 @@ http://wttv-lh.akamaihd.net:80/i/WTTVBreaking_1@333494/master.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="XXX",Eroxxx HD
+http://147.78.163.247:8000/play/a021
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/KOgoXoG.png" group-title="",MCN6
+https://d18fcxaqfnwjhj.cloudfront.net/CDN_Ingest/MCN6_MAIN.smil/Playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/KOgoXoG.png" group-title="",MCN6 Arts Channel
+https://d18fcxaqfnwjhj.cloudfront.net/CDN_Ingest/MCN6_COMEDY.smil/Playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/KOgoXoG.png" group-title="",MCN6 Music Channel
+https://d18fcxaqfnwjhj.cloudfront.net/CDN_Ingest/MCN6_MUSIC.smil/Playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",ASS TV
+http://www.ast.tv/stream/2/master.m3u8?fluxuslust.m3u8
```
Comment (freearhey) [inline channels/us.m3u:3582]: At the moment, links to AST TV channel are placed in the Cypriot playlist: https://github.com/iptv-org/iptv/pull/2146

File: channels/us.m3u [hunk 2]
```diff
@@ -3564,3 +3564,21 @@ http://wttv-lh.akamaihd.net:80/i/WTTVBreaking_1@333494/master.m3u8
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="Mohabat TV" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/tucTC1O.jpg" group-title="",Mohabat TV
 http://media.mohabat.tv:1935/live_transcoder/ngrp:mohabat.stream_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Ind TV USA
+https://t06858-lh.akamaihd.net/i/t06858a_1@719164/index_1500000_av-p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Jasmin channel 18+ HD
+http://109.71.162.112:1935/live/hd.jasminchannel.stream/chunklist_w1793148297.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Jasmin TV
+http://109.71.162.112:1935/live/hd.jasminchannel.stream/chunklist_w359453841.m3u8
```
Comment (freearhey) [inline channels/us.m3u:3572]: Jasmin TV channel, as far as I know, broadcasts from Luxembourg.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

**Comment by freearhey** [comment]
Closed due to lack of any activity.

File: channels/us.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:3582]: AST's legal data says the society behind it is Chypriot.

File: channels/us.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:3572]: + IP verification proves it : 
image

File: channels/us.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:3582]: image

However, the host is indeed American, and the IP comes from... the Netherlands. A complete mix of informations there.

Comment (LaneSh4d0w) [reply channels/us.m3u:3582]: aaaand.... the website's terms of services says they're Chypriot... 
image
image

**Pull request closed without merging**